back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/parsers/idea_pdf_file_parser.rb

```diff
@@ -150,6 +160,24 @@ def merge_pdf_rows(form_parsed_ideas, text_parsed_ideas, file)
       end
     end
 
+    def process_field_value(field, form_fields)
+      field = super field, form_fields
+
+      if %w[text multiline_text text_multiloc html_multiloc].include?(field[:input_type]) && field[:value]
```

Comment:
I also used this array in my PR https://github.com/CitizenLabDotCo/citizenlab/pull/7667/files#diff-25fcb5dcd733dc1acaabeafcf50b6fe33bc3b5f405df79c0f1d55f4cb30eda9eR10

In your case, can the same also happen for number fields?

Reply:
Yes good point, it could happen with number fields